woof-distro/x86_64/ubuntu/noble64/_00build.conf

```diff
@@ -0,0 +1,168 @@
+#
+#  persistent configuration options
+#
+#  see also DISTRO_SPECS DISTRO_PET_REPOS DISTRO_COMPAT_REPOS-*
+#
+#  **NOTE**: check the original file every once in a while
+#            settings might be added or removed...
+#
+
+# 2createpackages
+STRIP_BINARIES=no
+
+## Kernel tarballs repo URL for choosing/downloading kernel
+KIT_KERNEL_REPO_URL=http://distro.ibiblio.org/puppylinux/huge_kernels
+
+### Kernel tarball URL - avoid being asked questions about downloading/choosing a kernel
+#KERNEL_TARBALL_URL=http://distro.ibiblio.org/puppylinux/huge_kernels/huge-5.10-kernel-kit.tar.bz2
+
+## an array of generically named programs to send to the ADRIVE, FDRIVE, YDRIVE
+## ADRV_INC="abiword gnumeric goffice"
+ADRV_INC=""
+## YDRV_INC=""
+YDRV_INC=""
+## FDRV_INC="" #this one is very experimental and it's recommended to be left unset
+FDRV_INC="linux-firmware firmware-sof-signed intel-microcode amd64-microcode"
+
+## Include kernel-kit generated FDRIVE
+## set to yes or no or leave commented to be asked the question at build time
+#KFDRIVE=no
+
+## build devx? yes/no - any other value = ask
+BUILD_DEVX=yes
+
+## include devx SFS in ISO?
+DEVX_IN_ISO=no
+
+## symlink /bin, /sbin and /lib to their /usr counterparts, like Debian?
+USR_SYMLINKS=yes
+
+## build bdrv? yes/no
+BUILD_BDRV=yes
+
+## packages to build from source
+PETBUILDS="busybox aaa_pup_c disktype dmz-cursor-theme firewallstatus freememapplet geany gexec viewnior gtkdialog gtk_theme_flat_grey_rounded gtk_theme_polished_blue gtk_theme_gradient_grey gtk_theme_buntoo_ambience gtk_theme_stark_blueish gxmessage jwm l3afpad lxtask lxterminal mtpaint pa-applet powerapplet_tray rox-filer transmission xarchiver xcur2png xdelta xdg-puppy-jwm Xdialog xlockmore yad pmaterial_icons puppy_standard_icons puppy_flat_icons ram-saver connman-puppy connman-ui connman-gtk fixmenusd spot-pkexec notification-daemon-stub palemoon weechat osmo abiword gnumeric xournalpp mpv efilinux gparted deadbeef gmeasures fpm2 xpad gtkhash gdmap homebank grsync hardinfo uget gfnrename fsearch mtr epdfview"
+
+## GTK+ version to use when building packages that support GTK+ 2
+PETBUILD_GTK=3
+
+## Include the windows puppy installer LICK by Luke Lorimer aka <noryb009>
+LICK_IN_ISO=yes
+
+## compression method to be used (SFS files)
+#SFSCOMP='-comp xz -Xbcj x86 -b 512K'
+#SFSCOMP='-comp xz -Xbcj arm,armthumb -b 512K'
+#SFSCOMP='-comp gzip'
+#SFSCOMP='-noI -noD -noF -noX'
+#SFSCOMP='-comp xz -Xbcj x86 -b 256K -no-exports -no-xattrs'
+SFSCOMP='-comp zstd -Xcompression-level 19 -b 256K -no-exports -no-xattrs'
```

Comment:
I had this doubt (not any issue) as to why we use zstd compression while xz provides much more space saving.

I tested this earlier for Jammy64, devx saved 16MB, docx saved 0.2MB, main iso saved 65MB, nls saved 11.5MB.

Is there a specific reason why we use chose zstd in newer puppy versions (since I don't think fossa uses it).

Reply:
The difference in size is ~10% but zstd is many times faster to decompress. With xz, you get a slightly smaller Puppy, but applications start much slower, CPU consumption is higher and boot times are worse.

Reply:
zstd vs xz is a question of tradeoffs. Storage is easy and cheap to expand or upgrade, and most old computers can afford a 10% increase in the size of Puppy even without any hardware change, but the CPU can't be upgraded. Therefore, if I need to decide what to compromise on, it must be size and not CPU consumption. I think it's a shame many Puppy developers choose xz and blindly assume that smaller size equals better performance, because this is a very obvious case where this assumption is false.